DQMOffline/L1Trigger/python/L1TriggerDqmOffline_cff.py

```diff
@@ -19,7 +19,9 @@
 
 import DQMServices.Components.DQMEnvironment_cfi
 dqmEnvL1T = DQMServices.Components.DQMEnvironment_cfi.dqmEnv.clone()
-dqmEnvL1T.subSystemFolder = 'L1T'
+dqmEnvL1T.subSystemFolder = 'L1T2016'
```

Comment:
I believe we are dropping support for the legacy L1T DQM. Anyway, we were asked to populate the stage 2 L1T histograms into the L1T2016 folder.